src/ansys_sphinx_theme/search/fuse_search.py

```diff
@@ -147,6 +148,6 @@ def create_search_index(app, exception):
         search_index.build_sections()
         search_index_list.extend(search_index.indices)
 
-    search_index_path = app.builder.outdir / "_static" / "search.json"
+    search_index_path = Path(app.builder.outdir.resolve().absolute()) / "_static" / "search.json"
```

Comment:
AFAIK - -absolute returns a Path already
```suggestion
    search_index_path = app.builder.outdir.resolve().absolute() / "_static" / "search.json"
```

Reply:
Good point! I just realized that can't apply any of the Path methods because `app.builder.outdir` is a `str` on PyDPF side. Thus, I am casting it to a Path before applying the any of the `resolve()`, or `absolute()` methods.